esp-hal/src/dma/mod.rs

```diff
@@ -256,36 +284,43 @@ impl DmaDescriptor {
         next: core::ptr::null_mut(),
     };
 
-    fn set_size(&mut self, len: usize) {
+    /// Set the size of the buffer.
+    pub fn set_size(&mut self, len: usize) {
         self.flags.set_size(len as u16)
     }
 
-    fn set_length(&mut self, len: usize) {
+    /// Set the length of the descriptor
```

Comment:
Again I'm just picking nits, but please be consistent with your style and end sentences with peirods.

Reply:
Whoops, I meant to put periods everywhere, I missed a spot.